src/data/types.js

```diff
@@ -70,15 +70,30 @@
  * @property {string} name name of the test plan, defaults to 'unknown'
  * @property {AriaATCIData.Log[]} log debug messages emitted during execution of test plan
  * @property {object[]} tests
+ * @property {string} tests[].id id of a test in a test plan
  * @property {string} tests[].filepath filepath of file describing the test in the test plan
  * @property {AriaATCIData.Log[]} tests[].log subset of log emitted during this single test
- * @property {AriaATCIData.TestResult[]} tests[].results
+ * @property {AriaATCIData.TestResultOutput[]} tests[].results
  */
 
 /**
  * Result from a single test in a test plan.
  * @typedef AriaATCIData.TestResult
- * @property {number} testId numeric id of a test in a test plan
+ * @property {string} testId id of a test in a test plan
+ * @property {number} presentationNumber numeric id of a test in a test plan
```

Comment:
Note that in the previous version we were inserting the string id as `testId` even though the type  specifies that we were expecting a numeric id. I updated the types to reflect this. I'm assuming we want to stick with using the string version as the `id` and not the numeric `presentationNumber`...